_books/ion-1-1/src/modules/system_module.md

```diff
@@ -7,23 +7,20 @@ leverages the system symbol table, the text encoding that users typically intera
 The system macros are more visible, especially to authors of macros.
 
 This chapter catalogs the system-provided symbols and macros.
-The examples below use unqualified names, which works assuming no other macros with the same name are in scope. The unambiguous form `$ion::macro-name` is always available to use in the [template definition language](../macros/defining_macros.md#template-definition-language-tdl).
+The examples below use unqualified names, which works assuming no other macros with the same name are in scope.
+The unambiguous form `$ion::macro-name` is always available to use.
```

Comment:
Unimportant, take it if you like it:
```suggestion
The unambiguous form `$ion::macro_name` is always available to use.
```
Hyphens are illegal in macro names, which is arguably either a good thing (because it's clear `macro-name` is a placeholder) or a bad thing (because people might think kebab case is accepted.)